src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.page__catalog {
+  display: grid;
+  justify-content: center;
```

Comment:
I don't know, if GRID is good choice in this task. There was a tip, do not use a Gap for separating cards.